assets/antd-custom.less

```diff
@@ -1,2 +0,0 @@
-@color-primary: #008DA6;
-@link-color: #00A3BF;
```

Comment:
why is this deleted now?

Reply:
i was trying something out but it's better to have it back. reverted.